.github/workflows/promptflow-evals-installation-test.yml

```diff
@@ -0,0 +1,64 @@
+name: promptflow-evals-installation-test

```

Comment:
Can we use this chance to rename regression-test to performance-test? Also, I sort of remember that we already capture the installation time in regression test. Can we remove the code there if that's duplicated with the change here?

Reply:
Renamed and removed the installation -time code from performance test.